e2e/tests/manageCases/caseProgression/checkApplication/checkApplication.spec.ts

```diff
@@ -8,8 +8,10 @@ import config from "../../../../config";
 test.use({ storageState: Config.sessionStoragePath + "caseWorker.json" });
 
 test.describe("Check Application task for DA Citizen case tests. @manageCases", () => {
+  let ccdRef: string = "";
+
   test.beforeEach(async ({ page }) => {
-    const ccdRef: string = await createDaCitizenCourtNavCase(false);
+    ccdRef = await createDaCitizenCourtNavCase(false);
```

Comment:
`let ccdRef: string = await createDaCitizenCourtNavCase(false);`

Reply:
It doesn't need it because it needs to be put into the global empty string above `let ccdRef: string = "";`